modules/ztp-applying-source-custom-resource-policies.adoc

```diff
@@ -0,0 +1,159 @@
+/ Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-applying-source-custom-resource-policies_{context}"]
+= Applying source custom resource policies
+
+Source custom resource policies include:
```

Comment:
include the following: --> I've always been told to introduce a list with a complete sentence.

Reply:
Change made.